docs(refs T30969): add checkbox documentation (#64)
demos-europe/demosplan-ui: src/components/DpCheckbox/DpCheckbox.stories.mdx

import { Canvas, Meta, Story } from '@storybook/addon-docs'
import DpCheckbox from './DpCheckbox'

<Meta
  title="Components/Checkbox"
  component={DpCheckbox} />

# Checkbox

The `DpCheckbox` component renders an input of type `checkbox`. Use it for a single option that the user can turn on or
off.

> In order to display several checkboxes grouped together, use `DpCheckboxGroup`. It allows you to define a group
> label and have checkbox labels with normal `font-weight`.

Reviewer: ```suggestion
In order to display several checkboxes grouped together, use `DpCheckboxGroup`. 
It allows for defining an additional group label and have checkbox labels with normal `font-weight`.
```
